examples/aws-ml-app-demo/liminal.yml

```diff
@@ -1,22 +1,30 @@
 ---
 name: MyDataScienceApp
 owner: Bosco Albert Baracus
+volumes:
+  - volume: gettingstartedvol
+    claim_name: gettingstartedvol-pvc
+    local:
+      path: .
 images:
   - image: myorg/mydatascienceapp
     type: python
-    source: .
 services:
   - service:
     name: my_datascience_server
     type: python_server
     description: my ds server
     image: myorg/mydatascienceapp
+    source: .
     endpoints:
       - endpoint: /predict
         module: serving
         function: predict
+      - endpoint: /healthcheck
+        module: serving
+        function: healthcheck
 pipelines:
-  - pipeline: my_datascience_pipeline
+  - pipeline: my_datascience_server
```

Comment:
I'm confused by this change, why is `server` now part of the pipeline name?

Reply:
my bad